### PR TITLE
feat: adds OSX support

### PR DESCRIPTION
**Pull request by fdidron**
Thanks for the example, really helped me evaluate web view for a future app. This PR adds OSX compilation support.

### CODE REVIEW

**Comment by wilricknl** [comment]
Hey, I'm glad you like the project. Does the WebKit framework come preinstalled on MacOS? If not, could you specify how to install it?

**Comment by fdidron** [comment]
It comes preinstalled yes

**Review by wilricknl — COMMENTED** [review]
Looks great. I will update the README and CHANGELOG, then merge this feature later today. Thanks for the contribution!

File: README.md [hunk 1]
```diff
@@ -29,6 +32,17 @@ sudo apt-get install libwebkit2gtk-4.0-dev
...
+# MacOS
+
+## Requirements
+
+* CMake
+* WebKit framework (should come preinstalled)
+
+## Building and running
+
+Todo: instructions
```
Comment (wilricknl) [inline README.md:44]: @fdidron Would you mind adding how you build the project on MacOS here?

Comment (fdidron) [reply README.md:44]: @wilricknl Not sure how one would import the project into Xcode as I don't use it.

What I did from the projects root:
```
mkdir build
cd build
cmake ..
make
```

**Pull request merged**